crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -10,27 +10,30 @@ use std::path::PathBuf;
 pub(crate) struct TestContractCommand {
 	#[arg(short = 'p', long, help = "Path for the contract project [default: current directory]")]
 	path: Option<PathBuf>,
-	#[arg(short = 'f', long = "features", help = "Features for the contract project")]
+	#[arg(short = 'f', long = "features", help = "Features for the contract project", value_parser=["e2e-tests"])]
 	features: Option<String>,
+	#[arg(
+		short = 'n',
+		long = "contracts-node",
```

Comment:
Why does this have to be so wordy? The whole point of choosing pop (and all the shorter abbreviations for subcommands) was to make it as short to type as possible. Why cant it just be `--node`?

Reply:
Makes sense to keep it easier. Renamed to `--node`